crates/pallet-domains/src/staking.rs

```diff
@@ -542,21 +541,19 @@ pub(crate) fn do_convert_previous_epoch_withdrawal<T: Config>(
     };
 
     if let Some(WithdrawalInShares {
-        domain_epoch,
         unlock_at_confirmed_domain_block_number,
         shares,
         storage_fee_refund,
+        ..
```

Comment:
I'd prefer `domain_epoch: _`. Avoiding `..` ensures that you don't forget to handle new field if it is ever introduced in the future.